lib/react_client/react_interop.dart

```diff
@@ -411,6 +411,8 @@ class ReactDartInteropStatics2 implements ReactDartInteropStatics {
     )
         handleComponentDidUpdate,
     void Function(Component2 component) handleComponentWillUnmount,
+    void Function(Component2 component, dynamic error, dynamic info) handleComponentDidCatch,
+    void Function(ComponentStatics componentInstance, dynamic error) handleGetDerivedStateFromError,
```

Comment:
Is this `void`? I thought it was `JsMap`?

Reply:
Good catch, it was meant to be `JsMap`.